pkg/kube/util.go

```diff
@@ -27,6 +31,18 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+const (
+	// RemoteTokenDir defines the path where we store token from remote KUBECONFIG
+	RemoteTokenDir = "./var/run/secrets/remote-token"
+
+	//RemoteTokenFileName is the file name we used to store token from remote KUBECONFIG
+	RemoteTokenFileName = "remote-token"
+
+	//RemoteKubeConfigSecretName is the secret name for remote KUBECONFIG
+	RemoteKubeConfigSecretName = "istio-kubeconfig"
```

Comment:
I guess this is the kubeconfig for the master cluster. `Remote` is a little misleading

Reply:
this is the kubeconfig from remote config cluster